src/style.rs

```diff
@@ -68,7 +65,7 @@ impl ProgressStyle {
                 .collect(),
             progress_chars,
             char_width,
-            template: Cow::Borrowed("{wide_bar} {pos}/{len}"),
+            template: Box::<str>::from("{wide_bar} {pos}/{len}"),
```

Comment:
I'd prefer simplifying this to `"{wide_bar} {pos}/{len}".into()` (and the same for the other cases).